<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Material</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="accessibleName">
       <string/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Olympics Analysis</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Top 5 Gold medals in Tokyo Olympics</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QGridLayout" name="graphLayout1_1"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Is HomeAdvantage?</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <layout class="QGridLayout" name="graphLayout1_2"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Recent Olympics Results</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <layout class="QGridLayout" name="graphLayout1_3"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>with CORONA</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
          <item>
           <widget class="QGroupBox" name="graph2_3">
            <property name="title">
             <string>Corona Effected?</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QGridLayout" name="graphLayout2_3"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
            <item>
             <widget class="QGroupBox" name="graph2_1">
              <property name="title">
               <string>Confirmed Rate Barplot</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QGridLayout" name="graphLayout2_1"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="graph2_2">
              <property name="title">
               <string>Fatality Barplot</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QGridLayout" name="graphLayout2_2"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>with GDP</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>GDP Barchart</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <layout class="QGridLayout" name="graphLayout3_1"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>GDP Effected?</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QGridLayout" name="graphLayout3_2"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStyles">
    <property name="title">
     <string>Styles</string>
    </property>
   </widget>
   <addaction name="menuStyles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_vertical">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSubmenu_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submenu</string>
   </property>
  </action>
  <action name="actionSubmenu_3">
   <property name="text">
    <string>Submenu</string>
   </property>
  </action>
  <action name="actionSUBSUB">
   <property name="text">
    <string>SUBSUB</string>
   </property>
  </action>
  <action name="actionSUBSUB_2">
   <property name="text">
    <string>SUBSUB</string>
   </property>
  </action>
  <action name="actionSUBSUB_3">
   <property name="text">
    <string>SUBSUB</string>
   </property>
  </action>
  <action name="actiondissabled">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>dissabled</string>
   </property>
  </action>
  <action name="actionSubmenu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submenu</string>
   </property>
  </action>
  <action name="actionSubmenu_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Submenu</string>
   </property>
  </action>
  <action name="actionSubmenu_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Submenu</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="text">
    <string>Qt Material Theme</string>
   </property>
   <property name="toolTip">
    <string>Qt Material Theme</string>
   </property>
  </action>
  <action name="actionSelected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selected</string>
   </property>
  </action>
  <action name="actionaction">
   <property name="text">
    <string>action</string>
   </property>
  </action>
  <action name="actionaction2">
   <property name="text">
    <string>action2</string>
   </property>
  </action>
  <action name="actionaction3">
   <property name="text">
    <string>action3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
